src/vs/platform/editor/common/editor.ts

```diff
@@ -160,16 +160,10 @@ export enum EditorActivation {
 }
 
 export enum EditorResolution {
-
-	/**
-	 * Displays a picker and allows the user to decide which editor to use.
-	 */
-	PICK,
-
 	/**
 	 * Only exclusive editors are considered.
 	 */
-	EXCLUSIVE_ONLY
+	EXCLUSIVE_ONLY = 1
```

Comment:
If we only end up having one, should this be a boolean flag instead?

Reply:
Yeah I don't know if we ever plan to add another one. I could always make it a boolean flag and switch back after words if we add more.